runtime/runtime-params-estimator/emu-cost/io_cost.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+log=./io_log.txt
+base=.
+mark=`date +"%Y-%m-%d-%H-%M-%S"`
+
+echo $mark | tee -a $log
+
+# cargo build --release --package neard
+# cargo build --release --package genesis-populate
+
+#for acc in 10
+for acc in 1000000
```

Comment:
At 10M accounts we start to hit writes more without hitting reads, e.g.
```
ecuted 796777787 instructions; 3884188 bytes read; 3959013 bytes written
executed 796834434 instructions; 3884188 bytes read; 3959013 bytes written
executed 23042783600 instructions; 3884192 bytes read; 32135619 bytes written
executed 45075426756 instructions; 3884196 bytes read; 60371741 bytes written
```

Reply:
Interesting, although hard to properly interpret that.